packages/common/src/constants.ts

```diff
@@ -15,3 +15,14 @@ export const RUNNER_REGEX = /(\^?)(\d|x|\*)+\.(\d|x|\*)+\.(\d|x|\*)+/;
 export const DEFAULT_WORD_SIZE = 32;
 export const DEFAULT_LEAF = Buffer.from('0000000000000000000000000000000000000000000000000000000000000001', 'hex');
 export const MMR_AWAIT_TIME = 2;
+
+//DEPLOYMENT
+export const DEFAULT_DEPLOYMENT_TYPE = 'primary';
+export const INDEXER_V = 'v1.1.2';
+export const QUERY_V = 'v1.1.1';
+export const DEFAULT_ENDPOINT = 'wss://polkadot.api.onfinality.io/public-ws';
+export const DEFAULT_DICT_ENDPOINT = 'https://api.subquery.network/sq/subquery/polkadot-dictionary';
```

Comment:
currently these are just the default values, but user can use inputs to change them in accordance to their needs

Reply:
Do we have to provide default values? I think they can be removed as its very unlikely these values will be used

Reply:
It would be great if we could call https://api.thechaindata.com/doc/#/SubQuery/ProjectController_networkEndpoints and https://api.thechaindata.com/doc/#/SubQuery/ProjectController_networkDictionaries to suggest default endpoints.

And i think this is for versions https://api.thechaindata.com/doc/#/Info/InfoController_getImageVersions